### DISCUSSION TITLE
QoL - Check version mismatch between installed Biome version and JSON-schema version

### DISCUSSION THREAD

**Discussion by hornta** [biomejs/biome#3127] [Ideas]
When running Biome in version X and the version of the `$schema` property of `biome.json` is NOT X(https://biomejs.dev/schemas/Y/schema.json), it would be helpful with a little warning somewhere in case there are unexpected configuration changes.

**Comment by Conaclos** [comment]
We recommend running `biome migrate` when you update Biome. However, I accept that it could be helpful to have a warning.
